include/libtorrent/add_torrent_params.hpp

```diff
@@ -151,7 +151,7 @@ namespace libtorrent
 			// in this ``add_torrent_params`` object will take precedence over
 			// whatever states are saved in the resume data. For instance, the
 			// ``paused``, ``auto_managed``, ``sequential_download``, ``seed_mode``,
-			// ``super_seeding``, ``max_uploads``, ``max_connections``,
+			// ``super_seeding``, ``priority``, ``max_uploads``, ``max_connections``,
```

Comment:
It is a discribtion of overwrite resume data flag in add_torrent_params. My patch takes into account this flag. So if flag is specifed than priority will have default value. Do I understend the semantics of this flag correctly? Do you think I should exclude `priority` property from OverwriteResumeData - related logic?